Update "User Interface Configuration" documentation. (#9448)
geonetwork/core-geonetwork: docs/manual/docs/administrator-guide/configuring-the-catalog/user-interface-configuration.md

## General options

-   **Humanize dates**: check this box to show dates in a human-friendly format. If not set, the full date will be shown.

> -   **Date format**: Define the format for dates using Moment.js syntax (e.g. `DD-MM-YYYY`).

Reviewer: ```suggestion
-   **Date format**: Define the format for dates using [`Moment.js`-compatible syntax](https://momentjs.com/docs/#/parsing/string-format/) (e.g. `DD-MM-YYYY`).
```

-   **Home page**: Select this check box to determine whether the Logo and Link to the Home Page are visible in the top toolbar. If not set, no logo and link is shown.

> -   **Application URL**: Define the URL for the home page. In the majority of cases this can be left as the default.

Reviewer: ```suggestion
-   **Application URL**: Define the URL for the home page. It is recommended to keep the default value.
```

Also (see comments below): why should users not touch this. What will happen if they do?  Provide some minimal use case/ example.

-   **Search application**: Select this check box to determine whether the search application is visible in the top toolbar. If not set, no link is shown.

> -   **Application URL**: Define the URL for the search application. In the majority of cases this can be left as the default.

Reviewer: "In the majority of cases this can be left as the default." Why? Any use case?

-   **Map Application**: The initial checkbox allows the main map tab to be disabled. In this case there will be no map tab shown in the top toolbar but the mini map and extent map described above will still be visible.

> -   **Application URL**: This defines the URL for the map tab. In the majority of cases this can be left as the default.

Reviewer: "In the majority of cases this can be left as the default." Why? Any use case?

-   **Single tile WMS**: If enabled, WMS layers are requested as a single tile instead of multiple tiles.

> -   **User preference persistence**: This option determines the behaviour of cookies related to the map. The various options are listed below ('', 'sessionStorage', or 'localStorage').

Reviewer: ```suggestion
-   **User preference persistence**: This option determines the behaviour of cookies related to the map. The various options are listed below ('', 'sessionStorage', or 'localStorage'). If an empty string is set, user preferences will not be stored.
```

## Record View

-   **Show Social bar**: If enabled, the social media bar is enabled in record view.
-   **Is default contact view enabled**: If enabled, shows the default contact view for records.
-   **Show Status watermark**: Define for which record statuses a watermark should be displayed in the background of the record view.
-   **Show Status top bar**: Define for which record statuses a status bar should be displayed at the top of the record view.
-   **Show Citation**: If enabled, shows the record citation. You can also define a condition (JSON) for when to show it.
-   **Sort keywords alphabetically**: If enabled, keywords are sorted alphabetically in the record view.

> -   **Thesaurus keywords to highlight**: Choose thesaurus that you want keywords to be displayed as badges.

Reviewer: ```suggestion
-   **Thesaurus keywords to highlight**: Choose thesaurus for the keywords that you wish to be displayed as badges.
```
